edx_api/grades/models.py

```diff
@@ -28,18 +74,15 @@ def __init__(self, current_grade_list):
             self.current_grades[current_grade.course_id] = current_grade
 
     def __str__(self):
-        return "<Current Grades for user {username}>".format(username=self.username)
+        return "<Current Grades for user {username}>".format(
+            username=self.username
+        )
 
     @property
     def all_course_ids(self):
         """Helper property to return all the course ids of the current grades"""
         return self.current_grades.keys()
```

Comment:
Nit: as this is the same as `all_usernames` method, could we move this to a superclass?

Reply:
Same thing I think applies here as it does to `all_usernames`, since the property name is the important part here.